src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +26,7 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber++;
```

Comment:
This is a non static attribute binded to the instance and not the class. As such, this will leave all the instance with a sequence number of 1 and not achieve the functionality asked for in this activity.